compiler/rustc_abi/src/lib.rs

```diff
@@ -1895,3 +1898,18 @@ pub enum StructKind {
     /// A univariant, but with a prefix of an arbitrary size & alignment (e.g., enum tag).
     Prefixed(Size, Align),
 }
+
+#[derive(Clone, Debug, PartialEq, Eq, PartialOrd, Ord)]
+pub enum AbiFromStrErr {
+    /// not a known ABI
+    Unknown,
+    /// no "-unwind" variant
+    NoUnwind,
```

Comment:
Isn't this emitted when we *do* encounter an "unwind" variant? Seems odd that the error would then be called the opposite of what happened.^^

Reply:
Oh, yes, it is odd. It's an overly terse statement of "the unwind variant isn't the thing you should be feeding into this function". I'll rename it someways if this survives review.

Reply:
I think I rewrote it to be a bit better?

Reply:
Maybe call this variant `UnwindNotAllowed`?

Reply:
ah right I wound up unrewriting it when it became less necessary...